src/latex2sympy/latex2sympyAsUnit.py

```diff
@@ -0,0 +1,95 @@
+import sympy
+import sympy.physics.units as sympy_physics_units
+from latex2sympy.latex2sympy import LatexToSympy
+from latex2sympy.lib import LATEXLexerToken
+from latex2sympy.utils.json import has_type_or_token
+from latex2sympy.units import UNIT_ALIASES, PREFIX_ALIASES, find_unit, find_prefix, is_unit
+
+
+def process_sympy_as_unit(latex: str, variable_values: dict = {}):
+    instance = LatexToSympyAsUnit(latex, variable_values)
+    return instance.process_sympy()
+
+
+class LatexToSympyAsUnit(LatexToSympy):
+    def __init__(self, latex: str, variable_values: dict = {}):
+        super().__init__(latex, variable_values)
+
+    def process_sympy(self):
+        return_data = super().process_sympy()
+        if not is_unit(return_data):
+            raise Exception('Unrecognized unit')
+        return return_data
+
+    def convert_letters_to_postfix_list_items(self, atom_text):
+        # do not call parent class
+        new_list_items = []
+        if '\\: ' in atom_text:
+            atom_text_split = atom_text.split('\\: ')
+            for t in atom_text_split:
+                if t == '.':
+                    raise Exception('"." is an invalid symbol')
+                if len(t) > 0:
+                    new_list_items.append({'exp': {'comp': {'atom': {'atom_expr': {'text': t, 'type': LATEXLexerToken.LETTER.value}}}}})
+        return new_list_items
+
+    def convert_atom(self, atom):
+        if has_type_or_token(atom, LATEXLexerToken.LETTERS):
+            atom_text = atom.get('text')
+            unit = find_unit(atom_text)
+            if unit is not None:
+                return unit
+            # prefixes can be combined in `convert_postfix_list`
+            prefix = find_prefix(atom_text)
+            if prefix is not None:
+                return prefix
+            raise Exception('Unrecognized unit')
+
+        # fallback to parent class
+        return super().convert_atom(atom)
+
+    def get_atom_symbol_for_atom_expr(self, atom_name, type):
+        # by default use parent class
+        atom_symbol = super().get_atom_symbol_for_atom_expr(atom_name, type)
```

Comment:
@builtbywill Not really breaking anything, but this value isn't really being used by default (or at all) since the 'else' branch is returning or throwing.

Reply:
@tragessere ah yep, that makes sense. I just pushed a change to simplify the method. thanks!